react/components/EditorContainer/Sidebar/ComponentEditor/index.tsx

```diff
@@ -118,7 +118,6 @@ const ComponentEditor: React.FunctionComponent<Props> = ({
         >
           <Form
             formContext={{
-              addMessages: iframeRuntime.addMessages,
               isLayoutMode: editor.mode === 'layout',
               messages: iframeRuntime.messages,
```

Comment:
Do you still need the to pass the `messages` here?

Reply:
Nope. Thanks!